README.md

```diff
@@ -0,0 +1,132 @@
+# TABObserverSet
+
+`TABObserverSet`, originally conceived by [Mike Ash](https://github.com/mikeash/SwiftObserverSet), provides a Swift-y alternative to the traditional `NotificationCenter` style of reactive programming.
```

Comment:
provides a Swift-y interface? its not replacing Notification Center just wrapping the interface right?